tests/context/CarouselProvider.test.js

```diff
@@ -671,7 +671,7 @@ describe('<CarouselProvider/>', () => {
             expect(result).toBe(prevDirection);
         });
 
-        test('should return prev if new active item is the last item of items', () => {
+        test('should return next if new active item is the last item of items', () => {
```

Comment:
Please add isInfinite to the two test cases you edited. Since the other two above already check what you are checking in these